packages/react/src/use-render/useRender.ts

```diff
@@ -25,17 +24,19 @@ export function useRender<
     state: (state ?? emptyObject) as State,
     ref: refsArray,
     extraProps,
-    skipGeneratingStyleHooks: true,
+    styleHooks: false,
   });
 }
 
 export namespace useRender {
   export type RenderProp<State = Record<string, unknown>> =
     | ComponentRenderFn<React.HTMLAttributes<any>, State>
-    | React.ReactElement<Record<string, unknown>>
-    | keyof typeof defaultRenderFunctions;
+    | React.ReactElement<Record<string, unknown>>;
```

Comment:
Removed `keyof typeof defaultRenderFunctions` since we shouldn't allow strings

Reply:
Good call!